crux-test/test/crux/lubm_test.clj

```diff
@@ -40,15 +40,15 @@
 ;; This query bears large input and high selectivity. It queries about just one class and
 ;; one property and does not assume any hierarchy information or inference.
 (t/deftest test-lubm-query-01
-  (t/is (= #{[:http://www.Department0.University0.edu/GraduateStudent101]
-             [:http://www.Department0.University0.edu/GraduateStudent124]
-             [:http://www.Department0.University0.edu/GraduateStudent142]
-             [:http://www.Department0.University0.edu/GraduateStudent44]}
+  (t/is (= #{[(keyword "http://www.Department0.University0.edu/GraduateStudent101")]
+             [(keyword "http://www.Department0.University0.edu/GraduateStudent124")]
+             [(keyword "http://www.Department0.University0.edu/GraduateStudent142")]
+             [(keyword "http://www.Department0.University0.edu/GraduateStudent44")]}
            (api/q (api/db *api*)
                   (rdf/with-prefix {:ub "http://swat.cse.lehigh.edu/onto/univ-bench.owl#"}
                     '{:find [x]
                       :where [[x :rdf/type :ub/GraduateStudent]
-                              [x :ub/takesCourse :http://www.Department0.University0.edu/GraduateCourse0]]})))))
+                              [x :ub/takesCourse #=(keyword "http://www.Department0.University0.edu/GraduateCourse0")]]})))))
```

Comment:
Read-eval isn't necessairly a clean solution. It trades one dubious thing for another 😄

Maybe one alternative would be changing `http:` to `http`, making the keywords/symbols valid. I tried `#=` first as it doesn't change semantics.